test/runtests.jl

```diff
@@ -2,64 +2,41 @@ using Test
 using TOML, CSV, DataFrames, ADRIA
 using ADRIA.metrics: total_absolute_cover
 
-
 const ADRIA_DIR = pkgdir(ADRIA)
 const TEST_DATA_DIR = joinpath(ADRIA_DIR, "test", "data")
 const EXAMPLE_DOMAIN_PATH = joinpath(ADRIA_DIR, "examples", "Example_domain")
 
+function test_rs()
+    orig_loc = pwd()
+    this_loc = @__DIR__
+    cd(this_loc)
```

Comment:
This is now bad practice - use the constants defined above to

1. Replace the relative paths (Line 20 - 21) and use `joinpath` with `EXAMPLE_DOMAIN_PATH`
2. Remove all calls to `cd()`